frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx

```diff
@@ -295,7 +253,7 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = (props) => {
       fieldsForOpenAPI(props.openAPI).filter(
         (crdField) =>
           props.providedAPI.specDescriptors &&
-          !props.providedAPI.specDescriptors.some((d) => d.path === crdField.path),
+          !!props.providedAPI.specDescriptors.some((d) => d.path === crdField.path),
```

Comment:
Idk why this change is here, removing:
```suggestion
          !props.providedAPI.specDescriptors.some((d) => d.path === crdField.path),
```